ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/converter/PrefixAdderImpl.java

```diff
@@ -1,94 +1,22 @@
 package be.vlaanderen.informatievlaanderen.ldes.server.domain.converter;
 
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.collections.Prefixes;
 import org.apache.jena.rdf.model.Model;
-import org.apache.jena.rdf.model.Statement;
-import org.apache.jena.shared.PrefixMapping;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Optional;
+import java.util.List;
 
 @Component
 public class PrefixAdderImpl implements PrefixAdder {
+	private final List<Prefixes> prefixes;
 
-	private static final String VALID_LOCALNAME_REGEX = "([a-zA-Z][\\-_a-zA-Z0-9]*)$";
-	private static final String VALID_PREFIX_REGEX = "^[a-zA-Z_][\\w.-]*$"; // NCName regex
-	private static final Logger LOGGER = LoggerFactory.getLogger(PrefixAdderImpl.class);
+	public PrefixAdderImpl(List<Prefixes> prefixes) {
+		this.prefixes = prefixes;
+	}
```

Comment:
As far as i'm seeing, you're always passing an empty list to this contructor in production code.
If you want to have this filled in test code, i'd propose using a setter rather than misusing the constructor :)